homeassistant/components/swidget/coordinator.py

```diff
@@ -0,0 +1,54 @@
+"""Swidget Device Coordinator."""
+
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+from typing import Any
+
+from swidget import SwidgetDevice, SwidgetException
+
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.debounce import Debouncer
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+_LOGGER = logging.getLogger(__name__)
+REQUEST_REFRESH_DELAY = 1.0
+
+
+class SwidgetDataUpdateCoordinator(DataUpdateCoordinator[None]):
+    """DataUpdateCoordinator to gather data for a specific Swidget device."""
+
+    def __init__(
```

Comment:
Pass the config entry too, and pass that on to super

Reply:
Fixed